app/serialisers/errors_serializer.rb

```diff
@@ -6,6 +6,7 @@ def initialize(message:)
   end
 
   def attributes
+    Rails.logger.info(message)
```

Comment:
are we keeping this in?

Reply:
Better not. I was considering leaving a Rails.logger.debug there to be only on dev environment but I will remove.